models/issues/issue_update.go

```diff
@@ -104,30 +106,43 @@ func doChangeIssueStatus(ctx context.Context, issue *Issue, doer *user_model.Use
 
 	// New action comment
 	cmtType := CommentTypeClose
+	var content string
+	if !issue.IsPull && issue.IsClosed {
+		c := &ClosedIssueCommentContent{
+			Tr: issueClosedCommentTrMap[issue.ClosedStatus],
+		}
+		data, err := json.Marshal(c)
+		if err != nil {
+			return nil, err
+		}
+		content = string(data)
```

Comment:
Storing a translation key into database seems quite strange and fragile.

Could there be other solutions? eg: use the related translation key when rendering?

Reply:
72a1e78 now only the vlaue of close_status is stored in db.